### PR TITLE
[camera] Fix preview freeze issue

### PR DESCRIPTION
**Pull request by bbrto21**
Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

**Comment by bbrto21** [comment]
fix #268

**Review by bwikbs — APPROVED** [review]
:+1:

File: packages/camera/tizen/src/camera_device.cc [hunk 1]
```diff
@@ -38,8 +40,19 @@ static std::string CreateTempFileName(const std::string &prefix,
...
+  int ret = media_packet_get_tbm_surface(media_packet, &surface);
+  if (ret != MEDIA_PACKET_ERROR_NONE) {
+    return false;
+  }
+  return true;
+}
+
+}  // namespace
+
+ExifTagOrientation ChooseExifTagOrientatoin(OrientationType device_orientation,
```
Comment (HakkyuKim) [inline packages/camera/tizen/src/camera_device.cc:54]: Is there a reason for not including this in the anonymous namespace after removing `static`?

File: packages/camera/tizen/src/camera_device.cc [hunk 2]
```diff
@@ -79,7 +92,7 @@ static ExifTagOrientation ChooseExifTagOrientatoin(
   return orientation;
 }
 
-static RecorderOrientationTag ChooseRecorderOrientationTag(
+RecorderOrientationTag ChooseRecorderOrientationTag(
```
Comment (HakkyuKim) [inline packages/camera/tizen/src/camera_device.cc:95]: Ditto.

File: packages/camera/tizen/src/camera_device.cc (hunk 1 above)
Comment (bbrto21) [reply packages/camera/tizen/src/camera_device.cc:52]: Oh... I thought I'd need it somewhere else someday, now that I think about it, I don't think there's any need for that.
I wrapped the functions in anonymous namespaces :)

**Review by HakkyuKim — APPROVED** [review]
LGTM

**Comment by bbrto21** [comment]
published

**Pull request merged**